frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -224,8 +224,6 @@ class PublicOrEmbeddedDashboardInner extends Component<PublicOrEmbeddedDashboard
       dashcard => !isActionDashCard(dashcard),
     );
 
-    const downloadsEnabled = PLUGIN_RESOURCE_DOWNLOADS.areDownloadsEnabled();
```

Comment:
I had to move this to the hook used by DashboardControls because PublicOrEmbeddedDashboard is used by the sdk where we can't use the hash parameters.